maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -373,7 +378,48 @@ class IOSDriver(
     override fun backPress() {}
 
     override fun hideKeyboard() {
-        pressKey(KeyCode.ENTER)
+        swipe(
+            start = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.5)),
+            end = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.47)),
+            durationMs = 50,
+        )
+
+        if (isKeyboardHidden()) {
+            return
+        }
+
+        swipe(
+            start = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.5)),
+            end = Point(widthPercentToPoint(0.47), heightPercentToPoint(0.5)),
+            durationMs = 50,
+        )
+
+        if (isKeyboardHidden()) {
+            return
+        }
+
+        tapOnNonClickableText()
```

Comment:
Is tapping still needed if we implement the swipe solution?

Reply:
Yes, it's just there as a fallback if the swipe fails to hide the keyboard.